app/views/conference/schedule.html.erb

```diff
@@ -25,3 +25,19 @@
       <%= render 'all_events' %>
   <% end %>
 </div>
+
+<%= javascript_tag do %>
```

Comment:
No I mean not rendering everything in `ConferenceController#schedule` and switching between views in `schedule.html.erb` but having an separate method and view like `ConferenceController#all`.